### DISCUSSION TITLE
Feature: Time Capsule Issues

### DISCUSSION THREAD

**Discussion by eightnoteight** [community/community#64108] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

I started a new repository and got my first PR, and as a remembrance I want to create a time capsule github issue like #2 that mentions #10000.

that way when someone creates a #10000 issue or PR, the time capsule issue automatically gets mentioned and everyone will have a good laugh or feel nostalgic. hopefully the license doesn't change by that time. but unfortunately this doesn't work because currently the references only work for older github issues/PRs. please make references work for future issues/PRs as well

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by christophercouch** [comment]
Hey @eightnoteight,

That's a fun idea! Creating nostalgic markers like a "time capsule" issue is a unique way to celebrate milestones and reflect on the journey. Currently, GitHub doesn't have an automatic mention feature based on issue numbers, but it's an interesting suggestion. Thanks for sharing, and best of luck with your repository!
